packages/patch/patch.1.0.0/opam

```diff
@@ -8,8 +8,8 @@ bug-reports: "https://github.com/hannesm/patch/issues"
 license: "ISC"
 
 depends: [
-  "ocaml" {>= "4.04.2"}
-  "dune"
+  ("ocaml" {>= "4.04.2" & < "5.0"} | "ocaml" {>= "5.0"} & "base-bytes")
+  "dune" {>= "1.4.0"}
```

Comment:
Otherwise we're removing perfectly valid setup that existed before:
```suggestion
  ("dune" {>= "1.4.0"} | "dune" {< "1.4.0"} & "base-bytes")
```

Reply:
That's true - I hadn't considered transitive dependencies (i.e. `base-bytes` or `ocamlfind` being installed for other reasons)